apps/dashboard/src/app/layout.tsx

```diff
@@ -18,11 +18,15 @@ import { RouterProvider } from '@/components/router';
 import { CSPostHogProvider, UserIdentity } from './providers';
 
 export const metadata: Metadata = {
+  metadataBase:new URL("https://stack-auth.com/"),
```

Comment:
The base url should use the `DASHBOARD_BASE_URL` env var

Reply:
also please make sure the formatting is aligned with the other part of the project